flathub.json

```diff
@@ -0,0 +1,4 @@
+{
+  "only-arches": ["x86_64"],
+  "automerge-flathubbot-prs": true
```

Comment:
Automerging is a bad idea, imo .You aren't testing if anything breaks or regressions happened. Especially for something like a repackaged binary.

Someone has to deal with the fallout if a 'bad' build gets pushed to the users.

Reply:
> Automerging is a bad idea, imo .You aren't testing if anything breaks or regressions happened. Especially for something like a repackaged binary.
> 
> Someone has to deal with the fallout if a 'bad' build gets pushed to the users. Are there any technical reasons for using it?

Thanks for the advice. Yes it was a temporary choice. However this was not meant to be a stable release anyway, also this Snap repackage breaks very rarely. 
I'm pretty sure this Flatpak won't make it to the public anyway, I'm going to close my pull request in a couple days.